docs/library/theodore.md

```diff
@@ -0,0 +1,160 @@
+
+- [GPLv3](https://github.com/Zlika/theodore/blob/master/LICENSE)
+
+A summary of the licenses behind RetroArch and its cores have found [here](https://docs.libretro.com/tech/licenses/).
+
+## Requirements
+
+None
+
+## BIOS
+
+The Theodore core does not feature BIOS use.
+
+## Extensions
+
+Content that can be loaded by the Theodore core have the following file extensions:
+
+- .fd (floppy disk)
+- .sap (floppy disk)
+- .k7 (tape)
+- .rom (cartridge)
+- .m7 (cartridge)
+
+## Features
+
+Frontend-level settings or features that the Theodore core respects.
+
+| Feature           | Supported |
+|-------------------|:---------:|
+| Restart           | ✔         |
+| Saves             | ✕         |
+| States            | ✕         |
+| Rewind            | ✕         |
+| Netplay           | -         |
+| Core Options      | ✔         |
+| RetroAchievements | -         |
+| RetroArch Cheats  | -         |
+| Native Cheats     | -         |
+| Controls          | -         |
+| Remapping         | -         |
+| Multi-Mouse       | -         |
+| Rumble            | -         |
+| Sensors           | -         |
+| Camera            | -         |
+| Location          | -         |
+| Subsystem         | -         |
+| [Softpatching](https://docs.libretro.com/guides/softpatching/) | -         |
+| Disk Control      | -         |
+| Username          | -         |
+| Language          | -         |
+| Crop Overscan     | -         |
+| LEDs              | -         |
+
+### Directories
+
+The Theodore core's internal core name is 'theodore'
+
+**Frontend's System directory**
+
+| File         | Description |
+|:------------:|:-----------:|
+| theodore.cfg | Config file |
+
+### Geometry and timing
+
+- The Theodore core's base width is 672 pixels
+- The Theodore core's base height is 432 pixels
+
+## Usage
+
+![](../image/core/theodore/to8d_os.jpg)
+
+Click the top yellow rectangle on the right-hand side.
+
+## Core options
+
+The Theodore core has the following option(s) that can be tweaked from the core options menu. The default setting is bolded.
+
+- **Floppy write protection** [theodore_floppy_write_protect]  (disabled|**enabled**)
+
+- **Tape write protection** [theodore_tape_write_protect]  (disabled|**enabled**)
+
+- **Dump printer data to file** [theodore_printer_emulation] (**disabled**|enabled)
+
+## User 1 device types
+
+The Theodore core supports the following device type(s) in the controls menu, bolded device types are the default for the specified user(s):
+
+- None - Input disabled
+- **RetroPad** - Joypad
+- RetroPad w/ Analog - Joypad
+
+## Other devices
+
+- Light pen - The TO8 light pen inputs are mapped to the mouse
+
+## Joypad
+
+| RetroPad Inputs                        | User 1 input descriptors    |
+|----------------------------------------|-----------------------------|
+| ![](../image/retropad/retro_a.png)     | "Fire" button               |
+| ![](../image/retropad/retro_b.png)     | Start button                |
```

Comment:
It's not really a "start" button. It just simulates a keystroke on the 'B' key. The 'B' key is a shortcut to start the "BASIC 512", and most of the software must be started like this, so mapping the 'B' key on the 'B' button allows to start most of the games without the need for a real keyboard.

Reply:
Interesting. I did not know that BASIC 512 could automatically run bootable device.